lib/style.css

```diff
@@ -161,13 +161,16 @@ td.feed-source-info {
   width: 100%;
 }
 .feedLabel {
-  align-items: center;
   border-radius: 5px;
   border: 1px solid;
   height: 100%;
   max-width: 100%;
   min-width: 60px;
-  padding: 5px 0 8px 0;
+  overflow: hidden;
+  padding: 5px;
+  text-align: center;
```

Comment:
I think this is what causes the strange unequal padding that sort of pushes the text into the bottom right corner. I don't know of a way to get even alignment using only text rendering. To my knowledge, flexbox is needed.
<img width="440" alt="Screen Shot 2021-07-29 at 1 31 29 PM" src="https://user-images.githubusercontent.com/86619099/127538027-e5ed6a52-34a9-4bd8-8784-0f14a68fe3d5.png">

Reply:
I don't think padding works either, since now the icon would be affected along with the text (since there is no container)

Reply:
Try the changes from https://github.com/ibi-group/datatools-ui/pull/697/commits/e3dd6d8f8627e4a1dbaaa9e22e0139d79d0d8e9a.

Reply:
Also, a number of CSS rules have becomed orphaned with my changes, you should remove them if you choose to merge this PR.